### DISCUSSION TITLE
Cannot upload large file (>100MB)

### DISCUSSION THREAD

**Discussion by anthonny-gunawan** [minio/minio#15440] [Q&A] [answered]
Hello,

I ma trying to upload a big file to my bucket, but I was not able to do so via Browser, it give me "Error: Error - File size too large", I tried to search but I cannot seems to find where is this limit is set.
![image](https://user-images.githubusercontent.com/4499263/181916907-f0ed36ab-b114-4f3e-9ad6-acab281a6eb1.png)

I am running minio on docker, RELEASE.2022-07-06T20-29-49Z

Thanks in advance for the help

**Comment by harshavardhana** [comment, marked as answer]
Fix your proxy settings i.e nginx

**Reply by anthonny-gunawan** [reply]
Thanks! I tried directly accessing and it works. I did not actually use NGINX/ Proxy, but I did use the free Cloudfared tunnel, will need to figure out where is the limit in the chain.

**Reply by minecraftchest1** [reply]
Cloudflare has a 100MB upload cap on the free plan.

**Reply by ShreshthTiwari** [reply]
Its not 100MB upload cap its 100Mbps upload speed cap.

**Reply by PunkFleet** [reply]
> Its not 100MB upload cap its 100Mbps upload speed cap.

I'm still new to minio, but my usage scenario is consistent with cloudfalre as mentioned above, is there any way to break out of this limitation of cloudflare? It seems that the 100M value can't be changed on cloudflare, and even if I increase it to 500M using the business version, it doesn't seem to satisfy the uploading needs of other large files such as python packages.

**Reply by klauspost** [reply]
I suggest you reach out to Cloudflare with your issue.

**Reply by PunkFleet** [reply]
> I suggest you reach out to Cloudflare with your issue.

Cloudflare allows chunked file uploads on the requesting end, but the minio console doesn't seem to have that option.
